src/main/java/com/monumental/models/Monument.java

```diff
@@ -246,11 +243,13 @@ public String toString() {
     private String generateDescription() {
```

Comment:
This method is a bit of a mess right now. There's a ticket in our backlog to clean it up.